clean and update ffmpeg plugin (#1401)
OpenVisualCloud/SVT-AV1: README.md

  - Run the sample application to encode: `./SvtAv1EncApp -i [in.yuv] -w [width] -h [height] -b [out.ivf]`
  - Sample application supports reading from pipe. E.g. `ffmpeg -i [input.mp4] -nostdin -f rawvideo -pix_fmt yuv420p - | ./SvtAv1EncApp -i stdin -n [number_of_frames_to_encode] -w [width] -h [height]`

## SVT-AV1 ffmpeg plugin installation

> 

Reviewer: ```suggestion

These commands are specifically made for Linux using the bash shell, but it is possible to modify them to work with msys2 on windows and other setups.

```

Reviewer 2: would we be able to point to the actions scripts here to give more details ? or is it best to keep it general.

Reviewer: We could point to the action scripts if we want, an alternative is to use something similar to what we do in SVT-HEVC's doc https://github.com/OpenVisualCloud/SVT-HEVC/tree/master/ffmpeg_plugin#for-linux

### 1. Build and install SVT-AV1

``` bash

>    git clone --depth=1 https://github.com/OpenVisualCloud/SVT-AV1
>    cd SVT-AV1
>    cd Build
>    cmake .. -G"Unix Makefiles" -DCMAKE_BUILD_TYPE=Release
>    make -j $(nproc)
>    sudo make install

Reviewer: ```suggestion
git clone --depth=1 https://github.com/OpenVisualCloud/SVT-AV1
cd SVT-AV1/Build
cmake .. -DCMAKE_BUILD_TYPE=Release
cmake --build . --target install
# If you are on a non-windows system, you may need to use sudo for the last command
```

Although not a necessary change, the beginning tabs are not necessary and a few commands can be combined.

Using cmake and not specifying the `-G` option will also help cross-platform since it will let cmake choose the system's default generator

Reviewer 2: @1480c1 would the above changes cover the multi-threaded make execution ```make -j $(nproc)``` ?

Reviewer: Depending on the default generator, it would work if we added `--parallel $(nproc)` to the `cmake --build` line, but that is only available from cmake version 3.11 and forwards, else it will just use the generators default (nproc + 2 for Ninja, 1 for makefiles)

Reviewer 2: can we then keep ```make -j $(nproc)``` ?

Reviewer: Yeah, no need to change the last command

```

> ###  2. Enable libsvtav1 to FFmpeg

Reviewer: ```suggestion
###  2. Enable libsvtav1 in FFmpeg
```

wording nit

Reviewer 2: +1

> 

Reviewer: ```suggestion

If you wish to use an FFmpeg tag or release, you will need to use the patch file located in the 0.8.4 release under the ffmpeg_plugin folder.
To learn more on how to actually use it, please consult the readme.md from that folder.

```

You might want to mention if the person wishes to build using an FFmpeg tag to use the patch file under the SVT-AV1 0.8.4 tag as the patch file is no longer available in the latest revisions. No need to explicitly mention all of the steps as you can just point to the old ffmpeg_plugin/README.md file from the tag.

Reviewer 2: valid point that's the last milestone where the patch would work. +1

``` bash

>    git clone --depth=1 https://github.com/FFmpeg/FFmpeg ffmpeg
>    cd ffmpeg
>    export LD_LIBRARY_PATH+=":/usr/local/lib"
>    export PKG_CONFIG_PATH+=":/usr/local/lib/pkgconfig"
>    ./configure --enable-libsvtav1
>    make -j $(nproc)

Reviewer: ```suggestion
git clone --depth=1 https://github.com/FFmpeg/FFmpeg ffmpeg
cd ffmpeg
export LD_LIBRARY_PATH+=":/usr/local/lib" PKG_CONFIG_PATH+=":/usr/local/lib/pkgconfig"
./configure --enable-libsvtav1
make -j $(nproc)
```

same here, you might want to mention here that these are specifically supposed to run in a bash shell since the export and configure line will not work in any of window's default shell, and some of linux's other available shells.

Reviewer 2: might be best to keep them separate here to attract the user's attention to the pkgconfig dependency which we get many questions about

```

> ###  3. Verify

Reviewer: ```suggestion
###  3. Verify that ffmpeg is working
```

Reviewer 2: +1

``` bash

>    ./ffmpeg -video_size 720x480 -pixel_format yuv420p -f rawvideo -i input.yuv -c:v libsvtav1 -y test.mp4

Reviewer: ```suggestion
./ffmpeg -i input.mp4 -c:v libsvtav1 -y test.mp4
```

might be better to use a more commonly known format for a generic test command line since I'm not sure how many people are aware of what yuvs are outside of the video encoding and testing community and also why they would have to specify the other parameters.

Reviewer 2: +1 on this
